brokers/theia/broker.go

```diff
@@ -224,13 +224,11 @@ func AddPortToTooling(toolingConf *model.ToolingConf, pj *PackageJSON) {
 	toolingConf.WorkspaceEnv = append(toolingConf.WorkspaceEnv, model.EnvVar{Name: theiaEnvVar1, Value: theiaEnvVarValue})
 }
 
-// GetRndNumberAsString returns stringified random port from range 4000-6000
-func GetRndNumberAsString() string {
-	port := GetRndPort()
+func (b *Broker) getRndNumberAsString() string {
+	port := b.getRndPort() // CHANge name to something meaningful and/OR random
```

Comment:
`CHANge` -> `Change`?
What is name in comment?
`meaningful and/OR random` <- looks like assigned value is just random port. 

So, please make sure comment is up to date.